### PR TITLE
Add Signal.signame specs

### PR DESCRIPTION
**Pull request by ksss**
http://ruby-doc.org/core-2.3.1/Signal.html#method-c-signame

### CODE REVIEW

File: core/signal/signame_spec.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+
+describe "Signal.signame" do
+  it "takes a signal name with a well known signal number" do
+    Signal.signame(0).should == "EXIT"
+    Signal.signame(1).should == "HUP"
+    Signal.signame(2).should == "INT"
+  end
```
Comment (eregon) [inline core/signal/signame_spec.rb]: You should wrap this spec with:
```ruby
platform_is_not :windows do
```
as I think these numbers are POSIX-specific.

**Comment by eregon** [comment]
Looks good!
Can you add a guard so it also passes the Windows CI?

File: core/signal/signame_spec.rb (hunk 1 above)
Comment (eregon) [inline core/signal/signame_spec.rb]: It's also possible to use `Signal.list` to avoid hard-coding numbers.

Comment (ksss) [inline core/signal/signame_spec.rb]: I didn't want to depend on `Signal.list`.
But, It's not strong opinion.
I'll use `Signal.list`.

**Comment by ksss** [comment]
Finally, I wrote just `Signal.signame(0).should == "EXIT"`.
`0` is a special signal number on CRuby.
It works on all environment.

File: core/signal/signame_spec.rb (hunk 1 above)
Comment (eregon) [inline core/signal/signame_spec.rb]: Yes, it's a trade-off. If the numbers are portable on POSIX platforms, it's OK to use hardcoded numbers.

**Comment by eregon** [comment]
Thank you for your contribution!

**Pull request merged**